### DISCUSSION TITLE
installing home brew getting errors Help ! please

### DISCUSSION THREAD

**Discussion by bugacys** [Homebrew/discussions#3834] [Getting started]
Checking for `sudo` access (which may request your password)...
Password:
==> This script will install:
/opt/homebrew/bin/brew
/opt/homebrew/share/doc/homebrew
/opt/homebrew/share/man/man1/brew.1
/opt/homebrew/share/zsh/site-functions/_brew
/opt/homebrew/etc/bash_completion.d/brew
/opt/homebrew

Press RETURN/ENTER to continue or any other key to abort:
==> /usr/bin/sudo /usr/sbin/chown -R mercyokechukwu:admin /opt/homebrew
==> Downloading and installing Homebrew...
HEAD is now at c7f1a45ec Merge pull request #14046 from Bo98/unbottled-dependents
error: Not a valid ref: refs/remotes/origin/master
fatal: ambiguous argument 'refs/remotes/origin/master': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'

**Comment by gromgit** [comment]
https://docs.brew.sh/Common-Issues#installation-fails-with-unknown-revision-or-path-not-in-the-working-tree
